package.json

```diff
@@ -3,11 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.9.3",
+    "@emotion/styled": "^11.9.3",
+    "@mui/material": "^5.8.7",
```

Comment:
Where is @material-ui/core? Lack of this library in package.json cause errors when starting project
![image](https://user-images.githubusercontent.com/108327339/177725143-f3eb15e7-5f9a-43bd-a63c-291192913d22.png)